### ISSUE TITLE
I don't know why I get this bug

### ISSUE THREAD

**Issue by hckhanh** [oven-sh/bun#10911]
### How can we reproduce the crash?

Just install the bun as usual on Windows

### Relevant log output

```shell
Bun v1.1.4 (df190815) Windows x64
InstallCommand: dotenv bunfig extracted_packages(151)
Elapsed: 101733ms | User: 4593ms | Sys: 34250ms
RSS: 0.73GB | Peak: 0.77GB | Commit: 0.74GB | Faults: 342081
----- bun meta -----

0   00007FF7F9BB2198
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
9   ???
10  ???
11  ???
12  ???

Search GitHub issues https://bun.sh/issues or join in #windows channel in https://bun.sh/discord

thread 17028 panic: index out of bounds
???:?:?: 0x7ff7f88f64ea in ??? (bun.exe)
???:?:?: 0x7ff7f88f702c in ??? (bun.exe)
???:?:?: 0x7ff7f872906a in ??? (bun.exe)
???:?:?: 0x7ff7f8723e47 in ??? (bun.exe)
???:?:?: 0x7ff7f8719669 in ??? (bun.exe)
???:?:?: 0x7ff7f87154c1 in ??? (bun.exe)
???:?:?: 0x7ff7f87cfd4e in ??? (bun.exe)
???:?:?: 0x7ff7f84f1603 in ??? (bun.exe)
???:?:?: 0x7ff7fb0b4863 in ??? (bun.exe)
???:?:?: 0x7ff8ef8d257c in ??? (KERNEL32.DLL)
???:?:?: 0x7ff8f1aeaa47 in ??? (ntdll.dll)
```

### Stack Trace (bun.report)

Bun v1.1.7 ([`b0b7db5`](https://github.com/oven-sh/bun/tree/b0b7db5c060e830801b0273b7ff6e34e3eb5c254)) on windows x86_64 [InstallCommand]

**panic**: index out of bounds

- [`install.zig:4092`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/install/install.zig#L4092): `enqueueDependencyWithMain`
- [`install.zig:4932`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/install/install.zig#L4932): `processDependencyList`
- [`install.zig:5652`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/install/install.zig#L5652): `runTasks`
- [`install.zig:10135`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/install/install.zig#L10135): `installPackages`
- [`install.zig:10815`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/install/install.zig#L10815): `installWithManager`
- [`filter_run.zig:428`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/cli/filter_run.zig#L428): `install`
- [`cli.zig:1440`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/cli.zig#L1440): `start`
- [`cli.zig:62`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/cli.zig#L62): `start`
- [`main.zig:51`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/main.zig#L51): `main`
- [`exe_common.inl:78`](https://github.com/oven-sh/bun/blob/b0b7db5c060e830801b0273b7ff6e34e3eb5c254/src/src/startup/exe_common.inl#L78): `invoke_main`

**Comment by Jarred-Sumner** [comment]
Duplicate of #9808

**Comment by Jarred-Sumner** [comment]
Closing as duplicate, we will fix this but please follow along in #9808
